src/controllers/company.details.controller.ts

```diff
@@ -73,6 +80,16 @@ export const confirmCompanyStartRequest = async (req: Request, res: Response, ne
   }
 };
 
+const checkIsExtensionLimitReached = (company: ExtensionsCompanyProfile): boolean => {
+  const dueDate: Date = new Date(company.accountsDue);
```

Comment:
It looks as if 'accountsDue' can be empty - just discovered that the processor-api code certainly handles this eventuality. If it is I think that the extension isn't auto-rejected. This code *may* work in that scenario but it would be good to have some explicit unit-testing around it.

Reply:
Added a unit test.